recipe/meta.yaml

```diff
@@ -1,22 +1,22 @@
-{% set year = '2018' %}
-{% set version = year + '0712' %}
-{% set next_year = year|int + 1 %}
+{% set soname = '152' %}
+{% set release_date = '20180712' %}
+{% set version = soname + '.' + release_date %}
```

Comment:
The problem here is that 152 < 20180712. So conda will pickup the old packages probably.

Reply:
I have seen versions like `1:152.20180712` in arch linux. Will this work with conda?

Reply:
https://wiki.archlinux.org/index.php/PKGBUILD#epoch